safe/gui/widgets/dock.py

```diff
@@ -756,7 +758,16 @@ def show_print_dialog(self):
                 keywords = self.keyword_io.read_keywords(active_layer)
 
                 provenances = keywords.get('provenance_data', {})
-                if provenances:
+                extra_keywords = keywords.get('extra_keywords', {})
+                is_multi_exposure = (
+                    extra_keywords.get(
+                        'analysis_type') == MULTI_EXPOSURE_ANALYSIS_FLAG)
```

Comment:
Is-there not an extra keywords key that you can use, from definitions?

Reply:
I don't find any, and I think my purpose on creating this `extra_keywords` at first is for this

Reply:
Yes there is: https://github.com/inasafe/inasafe/blob/develop/safe/definitions/extra_keywords.py#L14

Reply:
ah you meant to use key from defintions, ok sure, missunderstanding :sweat_smile:

Reply:
Yes, using the key ;-)